lib/avoidance/include/avoidance.hpp

```diff
@@ -35,12 +35,15 @@
 #include "cogip_defs/Coords.hpp"
 #include "tracefd/File.hpp"
 
+#include "PB_Coords.hpp"
+
 /**
  * @name Dijkstra avoidance parameters
  * @{
  */
-#define GRAPH_MAX_VERTICES      64          /**< Maximum graph vertice number */
-#define DIJKSTRA_MAX_DISTANCE   UINT32_MAX  /**< Maximum Dijkstra distance */
+#define GRAPH_MAX_VERTICES         64          /**< Maximum graph vertice number */
+#define DIJKSTRA_MAX_DISTANCE      UINT32_MAX  /**< Maximum Dijkstra distance */
+#define AVOIDANCE_MAX_POINTS_PATH  32          /**< Maximum number of points in avoidance path */
```

Comment:
What is the purpose of AVOIDANCE_MAX_POINTS_PATH ???
And why 32 ?

Reply:
Used GRAPH_MAX_VERTICES instead.